package.json

```diff
@@ -208,7 +208,7 @@
     "balena-image-manager": "^7.1.1",
     "balena-preload": "^11.0.0",
     "balena-release": "^3.2.0",
-    "balena-sdk": "^16.8.1",
+    "balena-sdk": "^16.9.0",
```

Comment:
The practical effect of this SDK update should be to cause the `device os-update` command to use @thgreasi's improvements to HUP. (I have not actually tested the `device os-update` command.)